components/molecules/Breadcrumb.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import BreadcrumbMenu from './BreadcrumbMenu';
+
+const baseClass = 'gds-page-header__breadcrumbs-list-item';
+const filterMenu = pathname => ({ path }) => path.charAt(0) !== ':' && !pathname.includes(path);
+
+const Breadcrumb = props => {
+    const {
+        config: { title, path, subpaths },
+        linkComponent: LinkComponent,
+        pathname,
+        hideMenus,
+        isLast,
+        className
+    } = props;
+    const hasSubpaths = !hideMenus && subpaths && subpaths.length;
+    const menu = hasSubpaths && subpaths.filter(filterMenu(pathname));
+    const hasMenu = !!(menu && menu.length);
+    const rootClass = cx(baseClass, className, {
+        [`${baseClass}--has-menu`]: hasMenu
+    });
+
+    return (
+        <li className={rootClass}>
+            {hasMenu && <BreadcrumbMenu linkComponent={LinkComponent} menu={menu} path={path} />}
+            {isLast ? (
+                title
+            ) : (
+                <LinkComponent className="gds-page-header__breadcrumbs-link" to={path}>
+                    {title}
+                </LinkComponent>
+            )}
+        </li>
+    );
+};
+
+Breadcrumb.displayName = 'Breadcrumb';
+
+Breadcrumb.defaultProps = {
+    className: ''
+};
```

Comment:
I don't think this is necessary when using classnames/cx. `undefined` values should be omitted.

Reply:
done